src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -148,7 +148,7 @@ private function parseFile($file)
         $source = str_replace('http://www.w3.org/2001/xml.xsd', $location, $source);
 
         if (!@$dom->schemaValidateSource($source)) {
-            throw new InvalidResourceException(implode("\n", $this->getXmlErrors($internalErrors)));
+            throw new InvalidResourceException("File:$file\n".implode("\n", $this->getXmlErrors($internalErrors)));
```

Comment:
Let's be more verbose and say something like *The "file.xlf" is not a valid resource*.

Also, our coding standard favours single quotes, and sprintf should be used to create an exception message:

```php
throw new InvalidResourceException(sprintf('The "%s" is not a valid resource: ', $file, implode("\n", $this->getXmlErrors($internalErrors))));
```